src/middlewared/middlewared/plugins/smart.py

```diff
@@ -0,0 +1,33 @@
+from middlewared.schema import accepts, Dict, Int, List, Str
+from middlewared.validators import Email
+from middlewared.service import SystemServiceService
+
+
+class SmartService(SystemServiceService):
+
+    class Config:
+        service = "smartd"
+        service_model = "smart"
+        datastore_prefix = "smart_"
+
+    @accepts(Dict(
+        'smart_update',
+        Int('interval'),
+        Str('powermode', enum=['never', 'sleep', 'standby', 'idle']),
```

Comment:
I had the idea of making an enums uppercase across middleware, for consistency. Do you think its a bad idea?